src/main/java/edu/ucla/library/bucketeer/utils/AnnotationMappingStrategy.java

```diff
@@ -19,6 +19,11 @@
 import edu.ucla.library.bucketeer.Constants;
 import edu.ucla.library.bucketeer.MessageCodes;
 
+/**
+ * A strategy for handling CSV mapping annotations.
+ *
+ * @param <T> A type to be map to CSV format
```

Comment:
Semi-recovered English major asks: shouldn't that be "a type to be mapped to"?

Reply:
Heh, as a semi-recovered English major myself, these sorts of questions are always welcome :-)